src/reference/asciidoc/appendix.adoc

```diff
@@ -659,4 +659,4 @@ Adds support for Dead Letter Exchanges/Dead Letter Queues.
 
 Adds an option to support adding a message id to logged messages.
 
-Adds an option to allow the specification of a `Charset` name to be used when converting `String`s to `byte[]`.
+Adds an option to allow the specification of a `Charset` name to be used when converting ``String``s to `byte[]`.
```

Comment:
This is how it looks in the PDF variant:
![pdf](https://cloud.githubusercontent.com/assets/1071590/22653580/dbc7c5ba-ec57-11e6-9530-fa826297ac15.png)

and HTML:
![html](https://cloud.githubusercontent.com/assets/1071590/22653726/4f782ea0-ec58-11e6-8251-fad2f5b2067a.png)



So, I'm afraid we can't accept this change.

In other places we use workaround like "`String` s".
In other words a whitespace before that `s` for plural.

I will merge it when you fix that way.
Thanks